frontend/src/routes/candidate/(protected)/questions/all/+page.svelte

```diff
@@ -0,0 +1,121 @@
+<script lang="ts">
+  import {page} from '$app/stores';
+  import {Button} from '$lib/components/button';
+  import {BasicPage} from '$lib/templates/basicPage';
+  import {Expander} from '$lib/components/expander';
+  import LikertResponseButtons from '$lib/components/questions/LikertResponseButtons.svelte';
+  import {answerContext} from '$lib/utils/answerStore';
+  import {goto} from '$app/navigation';
+  import Icon from '$lib/components/icon/Icon.svelte';
+  import {_} from 'svelte-i18n';
+
+  /**
+   * A small delay before moving to the unaswered question.
+   * TODO: Make this a global variable used throughout the app.
+   */
+  const DELAY_M_MS = 350;
+
+  const questions = $page.data.questions;
+  const categories = [...new Set(questions.map((question) => question.category))];
+
+  const store = answerContext.answers;
+  $: answerStore = $store;
+
+  let nofUnasweredQuestions = 0;
+  $: nofUnasweredQuestions = questions.length - Object.entries(answerStore).length;
+
+  function gotoToQuestion(id: string) {
+    const questionUrl = $page.url.pathname.replace(/\/all$/, '/' + id);
```

Comment:
This seems suspicious. I think the pathname could just be `/candidate/questions/${id}`